packages/flame/test/cache/assets_cache_test.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flame/cache.dart';
+import 'package:flame/flame.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+
+import '../fixtures/fixture_reader.dart';
+
+class _AssetsCacheMock extends Mock implements AssetsCache {
+  int disposedCount = 0;
+
+  void dispose() {
+    disposedCount++;
+  }
+}
```

Comment:
```suggestion
class _AssetsCacheMock extends Mock implements AssetsCache {}
```

Then you can verify the call could by using something like this:

```dart
verify(mock.dispose).called(2);
```